ci-operator/step-registry/ipi/conf/external-lb/ipi-conf-external-lb-commands.sh

```diff
@@ -0,0 +1,263 @@
+          exit 1
+        fi
+    fi
+fi
+
+# Load array created in setup-vips:
+# 0: API
+# 1: Ingress
+declare -a vips
+mapfile -t vips < "${SHARED_DIR}"/vips.txt
+
+hosted_zone_id="$(aws route53 list-hosted-zones-by-name \
+            --dns-name "${base_domain}" \
+            --query "HostedZones[? Config.PrivateZone != \`true\` && Name == \`${base_domain}.\`].Id" \
+            --output text)"
+echo "${hosted_zone_id}" > "${SHARED_DIR}/hosted-zone.txt"
+
+if [ "${JOB_NAME_SAFE}" = "launch" ]; then
+  # If this is a `launch` and we are in the multi-zone range, we also
+  # 
+  if [ $((${LEASED_RESOURCE//[!0-9]/})) -ge 151 ]; then
+    if [ $((${LEASED_RESOURCE//[!0-9]/})) -le 154 ]; then
+    # Configure DNS direct to respective VIP
+    api_dns_target='"TTL": 60,
+      "ResourceRecords": [{"Value": "'${vips[0]}'"}, {"Value": "169.48.190.22"}]'
+    apps_dns_target='"TTL": 60,
+      "ResourceRecords": [{"Value": "'${vips[1]}'"}, {"Value": "169.48.190.22"}]'
+    fi
+  else   
+    # Configure DNS target as previously configured NLB
+    nlb_arn=$(<"${SHARED_DIR}"/nlb_arn.txt)
+    nlb_dnsname="$(aws elbv2 describe-load-balancers \
+              --load-balancer-arns ${nlb_arn} \
+              --query 'LoadBalancers[0].DNSName' \
+              --output text)"
+    nlb_hosted_zone_id="$(aws elbv2 describe-load-balancers \
+              --load-balancer-arns ${nlb_arn} \
+              --query 'LoadBalancers[0].CanonicalHostedZoneId' \
+              --output text)"
+
+    # Both API and *.apps pipe through same NLB
+    api_dns_target='"AliasTarget": {
+          "HostedZoneId": "'${nlb_hosted_zone_id}'",
+          "DNSName": "'${nlb_dnsname}'",
+          "EvaluateTargetHealth": false
+          }'
+    apps_dns_target=$api_dns_target
+  fi 
+
+else
+  # Configure DNS direct to respective VIP
+  api_dns_target='"TTL": 60,
+        "ResourceRecords": [{"Value": "'${vips[0]}'"}]'
+  apps_dns_target='"TTL": 60,
+        "ResourceRecords": [{"Value": "'${vips[1]}'"}]'
+fi
+
+# api-int record is needed just for Windows nodes
+# TODO: Remove the api-int entry in future
+echo "Creating DNS records..."
+cat > "${SHARED_DIR}"/dns-create.json <<EOF
+{
+"Comment": "Create public OpenShift DNS records for VSphere IPI CI install",
+"Changes": [{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "api.$cluster_domain.",
+      "Type": "A",
+      $api_dns_target
+      }
+    },{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "api-int.$cluster_domain.",
+      "Type": "A",
+      "TTL": 60,
+      "ResourceRecords": [{"Value": "${vips[0]}"}]
+      }
+    },{
+    "Action": "UPSERT",
+    "ResourceRecordSet": {
+      "Name": "*.apps.$cluster_domain.",
+      "Type": "A",
+      $apps_dns_target
+      }
+}]}
+EOF
+
+# api-int record is needed for Windows nodes
+# TODO: Remove the api-int entry in future
+echo "Creating batch file to destroy DNS records"
+
+cat > "${SHARED_DIR}"/dns-delete.json <<EOF
```

Comment:
Here just only update existing api/ingress dns records, I guess they will be deleted when calling step [ipi-deprovision-vsphere-dns](https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/deprovision/vsphere/dns/ipi-deprovision-vsphere-dns-commands.sh#L33). So do we need to overwrite file `"${SHARED_DIR}"/dns-delete.json` here?